service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -1,23 +1,94 @@
 package uk.gov.hmcts.reform.fpl.model;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Builder;
 import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.Document;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentSocialWorkOther;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.EPOGroup;
 
 import java.util.List;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
 @Data
-@Builder(toBuilder = true)
-@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
 public class CaseData {
-    private final List<Element<Applicant>> applicants;
+    @NotBlank(message = "Enter a case name")
+    private final String caseName;
+    private final String gatekeeperEmail;
+    private final String caseLocalAuthority;
+    private final Risks risks;
+    @NotNull(message = "You need to add details to orders and directions needed")
+    @Valid
+    private final Orders orders;
+
+    @NotNull(message = "You need to add details to grounds for the application", groups = EPOGroup.class)
+    @Valid
+    private final Grounds grounds;
+
+    @NotNull(message = "You need to add details to grounds for the application",
+        groups = EPOGroup.class)
```

Comment:
Can we keep it in one line? Wrapping it manual does not necessarily make it more readable.

Also sometimes you have blank lines between properties, sometimes you do not. Would be nice to have some consistency there.

Reply:
Agreed to tidy up white space